doc/source/io.rst

```diff
@@ -20,6 +20,40 @@ For example:
 
     Screen.wrapper(demo)
 
+You can also use the :py:obj:`.ManagedScreen` class either as a function decorator or the `with` keyword:
+
+1. Use it as a method decorator. Note that I use `screen=None` in this example to prevent linting errors:
```

Comment:
Oops, I should have only left the note on this line

Reply:
Don't worry - I'll tweak it now.  That'll teach me to look more closely at the examples in the future!